openmdao/core/total_jac.py

```diff
@@ -228,6 +228,11 @@ def __init__(self, problem, of, wrt, use_abs_names, return_format, approx=False,
 
         abs2meta_out = model._var_allprocs_abs2meta['output']
 
+        for name, meta in responses.items():
+            if name not in abs2meta_out and meta['path'] in abs2meta_out and \
+                    meta['path'] is not None:
+                abs2meta_out[name] = abs2meta_out[meta['path']]
```

Comment:
model._var_allprocs_abs2meta is a dict of absolute continuous variable names that is used is various places in the framework for things like iteration and should never have anything added to it by total_jac.py.